Ontology/Capability/Sensor/ConductivitySensor.json

```diff
@@ -1,5 +1,5 @@
 {
-  "@id": "dtmi:digitaltwins:rec_3_3:device:MotionSensor;1",
+  "@id": "dtmi:digitaltwins:rec_3_3:device:ConductivitySensor;1",
```

Comment:
Are we removing MotionSensor?

Reply:
No, it has been moved down into the "BinarySensor" hierarchy, which simplifies typing of the "lastValue" property for all of those sensors that return binary values. If you think this is an inappropriate clustering, it is not a dealbreaker for me to revert.